manifests/dev/vsphere-7.0u2/vanilla/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -123,6 +123,9 @@ spec:
             - "--kube-api-qps=100"
             - "--kube-api-burst=100"
             - "--leader-election"
+            # needed only for topology aware setup
+            #- "--feature-gates=Topology=true"
```

Comment:
Is this feature gate needed for `v2.0.0`? I think its GA'd with this release of external-provisioner..

Reply:
yes, we still need it. The default is false.

https://github.com/kubernetes-csi/external-provisioner/blob/master/pkg/features/features.go#L35-L39

```
// defaultKubernetesFeatureGates consists of all known feature keys specific to external-provisioner.
// To add a new feature, define a key for it above and add it here.
var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
	Topology: {Default: false, PreRelease: featuregate.Beta},
}
```